tests/bugs/spirv-opt-SROA-of-globals.slang

```diff
@@ -0,0 +1,130 @@
+//TEST:SIMPLE(filecheck=CHECK): -target spirv -fvk-use-entrypoint-name
+
+// Note: SROA will only run for 100 elements if spirv-opt does not manually set the spirv-opt `scalar-replacement` option
+//CHECK-NOT: OpCompositeConstruct
+struct Data 
+{
+uint data0;
```

Comment:
will `uint data[100]` work?

Reply:
`uint data[100]` is treated as 1 element in the struct. If `uint data[100]` is alone, SROA will run and hoist members outside the struct. If `uint data[100]` is with 100 other elements, SROA will not run and a composite will be constructed.